### DISCUSSION TITLE
Database Encryption?

### DISCUSSION THREAD

**Discussion by Shadowfita** [appwrite/appwrite#7517] [Q&A]
Just wondering if there is database encryption available yet? I've seen mentions of it on the site in the docs, and some API requests including encryption as a boolean.

**Comment by MeisterDerMagie** [comment]
I also wonder about the details. In the documentation it says: "Databases will provide the ability to create encrypted columns. This feature is being actively developed and coming soon. Attributes will be encrypted with AES-128 in Galois/Counter Mode (GCM)."
Is there any official statement of what _coming soon_ means? Are we talking about a few months or a few years?

**Reply by eldadfux** [reply]
We have a non-official POC in the making, no ETA yet, not years if all goes to plans.

**Reply by MeisterDerMagie** [reply]
Glad to hear. This is an important feature to be GDPR-compliant.
Thanks for your awesome work!

**Reply by DH-555** [reply]
I think this feature is not strictly needed for GDPR compliance in most cases, as the data processing will depend on your privacy policy.

**Reply by MeisterDerMagie** [reply]
We plan to develop an educational app for schools, so we're working with children's personal data. I guess Article 32(1)(a) GDPR is relevant here which states that "the pseudonymisation and encryption of personal data" might be appropriate. Anyways, I'm glad you have the encryption on the radar. Thank you!
